.github/ISSUE_TEMPLATE/performance.yml

```diff
@@ -0,0 +1,136 @@
+name: Performance Issue Report
+description: This report is for the performance-related issue
+title: "[Performance]: "
+labels: ["performance", "support_request"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please provide all the necessary information to expedite the response.
+  - type: input
+    id: ov_version
+    attributes:
+      label: OpenVINO Version 
+      description: OpenVINO version, branch, or tag in OpenVINO GitHub
+      placeholder: 2021.4.0 LTS / Master Branch / tag 2022.3.0
+    validations:
+      required: false
+  - type: input
+    id: target_platform
+    attributes:
+      label: Target Platform
+      description: The platform that the OpenVINO source is built on
```

Comment:
Please add `log of "lscpu" and "lscpu -e"` for Linux

Reply:
Added in the description, so that customer can run the command and copy-paste.